templates/public/index.html

```diff
@@ -6,6 +6,7 @@
 <link rel="alternate" type="application/rss+xml" title="Arch Linux News Updates" href="/feeds/news/" />
 <link rel="alternate" type="application/rss+xml" title="Arch Linux Package Updates" href="/feeds/packages/" />
 <script type="text/javascript" src="{% static "homepage.js" %}" defer></script>
+{% if MASTODON_LINK %}<link rel="me" href="{{MASTODON_LINK}}"> {% endif %}
```

Comment:
maybe we want a useful title as well?

PS: For consistency it would make sense to close <link/> and also get rid of the trailing whitespace:

`{% if MASTODON_LINK %}<link rel="me" href="{{MASTODON_LINK}}" title="Arch Linux Mastodon" />{% endif %}`